src/screens/PinScreen.js

```diff
@@ -192,13 +192,26 @@ class PinScreen extends React.Component {
         return;
       }
 
-      const pinCorrect = cryptoUtils.checkPassword(accessData.words, pin);
+      let wordsEncryptedData = accessData.words
+      if (!accessData.words.data) {
```

Comment:
In the old format, `words` is a string, but in the new one it is an object, so we need to create the encrypted data object from the data present on the access data.